code-challenges.test.js

```diff
@@ -25,35 +38,140 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
+describe("divByThree", () => {
+    it("takes a object as an argument and decides if the number inside it is evenly divisible by three or not.", () => {
+    const object1 = { number: 15 }
+    const object1OutPut = "15 is divisible by three"
+    expect(divByThree(object1.number)).toEqual(object1OutPut)
```

Comment:
An important thing to note in your tests, you don't want to perform any logic or data manipulation inside here.  That should be reserved for the function.